src/app/models/color.ts

```diff
@@ -1,12 +1,21 @@
 import { RandomUtils } from '../utils/random.utils';
 
 export class Color {
-  constructor(public hexCode: string = '') {
-    if (!this.hexCode) {
-      this.hexCode = this.generateRandomHex();
+  //renamed to privateHexCode so it doesn't conflict with the setter
+  constructor(private privateHexCode: string = '') {
```

Comment:
Could you rename `privateHexCode` to `_hexCode`? Using underscores is the "standard" in these kinds of situations

Reply:
Made the change,
It was my original intention to name it that, but it breaks one of the eslint rules
`Unexpected dangling '_' in '_hexCode'.`

So I added `/* eslint-disable no-underscore-dangle */`

Is that acceptable?

Reply:
Yes, for sure! I don't quite agree with that rule.